src/app/app.jsx

```diff
@@ -94,7 +94,13 @@ export function App({ language, onChangeLanguage }) {
       <Breadcrumbs />
 
       <Switch>
-        <Route exact path={PATHNAME_MAIN_PAGE} component={MainPage} />
+        <Route exact path={PATHNAME_MAIN_PAGE}>
+          {getConfig().themeNap ? (
+            <Redirect to={PATHNAME_DATASETS} />
+          ) : (
+            <MainPage />
+          )}
+        </Route>
```

Comment:
Kunne det være?

```
{getConfig().themeNap
  ? <Redirect to={PATHNAME_DATASETS} />
   : <Route exact path={PATHNAME_MAIN_PAGE} component={MainPage} />
}
```

Jeg finner ikke dokumentasjon på bruk av children. Fleste steder brukes component prop.

Reply:
Jeg tror ikke jeg kan gjøre sånn for ønsker en redirect kun på rot-path ("/"), hvis config === nap. Tok utgangspunkt i dette eksempelet: (https://reactrouter.com/web/api/Redirect)
<Route exact path="/">
  {loggedIn ? <Redirect to="/dashboard" /> : <PublicHomePage />}
</Route>

Reply:
Cool! Det visste jeg ikke om